src/controllers/places/delete_place.js

```diff
@@ -0,0 +1,20 @@
+const Request = require('request');
+const { placesURL } = require('../../constants/urls.json');
+
+module.exports = (req, res) => {
+  const reqOptions = {
+    url: `${placesURL}/${req.params.id}`,
+    method: 'DELETE'
+  };
+
+  Request(reqOptions, (error, response) => {
+    if (error) {
+      res.send('huge failure');
```

Comment:
sorry, but lets still send a 404 here (like line 15)

Reply:
Should be a 500 yo